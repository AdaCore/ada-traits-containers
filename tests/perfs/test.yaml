title: 'maps'
description: 'Basic test for maps'
driver: 'build_and_exec'
pre: ['python ./generate_test.py']
manual: true
