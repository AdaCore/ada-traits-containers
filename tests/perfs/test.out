Run int_list_c++
Run str_list_c++
Run int_vector_c++
Run int_vector_ada_arrays
Run str_vector_c++
Run intint_map_c++_unordered
Run intint_map_c++
Run strstr_map_c++_unordered
Run strstr_map_c++
Run ada12_bounded
Run ada12_def_unbounded
Run ada12_indef_unbounded
Run ada12_nocheck_def_unbounded
Run indef_unbounded
Run def_unbounded
Run def_bounded
Run indef_unbounded_spark
Run ada12_indef_unbounded
Run ada12_nocheck_indef_unbounded
Run indef_unbounded
Run def_unbounded_string
Run indef_unbounded_spark
Run ada12_bounded
Run ada12_def_unbounded
Run ada12_indef_unbounded
Run ada12_nochecks_definite_unbounded
Run indef_unbounded
Run def_unbounded
Run def_bounded
Run indef_unbounded_spark
Run ada12_indef_unbounded
Run ada12_nochecks_indef_unbounded
Run indef_unbounded
Run indef_unbounded_spark
Run ada12_ordered_def_unbounded
Run ada12_hashed_def_unbounded
Run ada12_hashed_def_bounded
Run hashed_def_def_unbounded
Run hashed_linear_probing_def_def_unbounded
Run ada12_ordered_indef_unbounded
Run ada12_hashed_indef_unbounded
Run hashed_indef_indef_unbounded
Run hashed_linear_probing_indef_indef_unbounded
Run hashed_indef_indef_unbounded_spark
Run graph_c++
Run graph_ada_custom
Run graph_ada_adjacency_list
Open file://<pwd>/perfs//index.html
